src/AppBundle/Form/Type/DateTimePickerType.php

```diff
@@ -30,9 +30,9 @@ class DateTimePickerType extends AbstractType
 {
     private $formatConverter;
 
-    public function __construct()
+    public function __construct(MomentFormatConverter $converter)
     {
-        $this->formatConverter = new MomentFormatConverter();
+        $this->formatConverter = $converter;
```

Comment:
Hey, this is a cool benefit of the new system :). There is no longer any extra work to do things correctly :)